src/main/java/org/broadinstitute/consent/http/db/DatasetAssociationDAO.java

```diff
@@ -35,7 +35,7 @@ public interface DatasetAssociationDAO extends Transactional<DatasetAssociationD
     @SqlQuery("select exists (select * from dataset_user_association where datasetId = :datasetId )")
     Boolean exist(@Bind("datasetId") Integer datasetId);
 
-    @SqlQuery("select ds.dacuserId from dataset_user_association ds where ds.dataset_id = :datasetId")
+    @SqlQuery("select ds.dacuserId from dataset_user_association ds where ds.datasetId = :datasetId")
```

Comment:
This snuck in in a previous PR but I don't think it was intentional: [Standardize DatasetDAO](https://github.com/DataBiosphere/consent/pull/1706/files#diff-7c59681ef9f69e77ecae62ac17e29ce35c775d05a0633db76932cb4f021a5713R38).

Reply:
I think you're right, nice catch